pylsp/workspace.py

```diff
@@ -625,6 +626,28 @@ def cell_data(self):
             offset += num_lines
         return cell_data
 
+    @lock
+    def jedi_names(
+        self,
+        up_to_cell_uri: Optional[str] = None,
+        all_scopes=False,
+        definitions=True,
+        references=False,
+    ):
+        """
+        Get the names in the notebook up to a certain cell.
+
+        :param up_to_cell_uri: The cell uri to stop at. If None, all cells are considered.
```

Comment:
```suggestion
        Parameters
        ----------
        up_to_cell_uri: str, optional
            The cell uri to stop at. If None, all cells are considered.
```

We prefer Numpy style docstrings.